packages/devtools_app/lib/src/shared/editor/api_classes.dart

```diff
@@ -442,6 +442,56 @@ class EditableArgumentsResult with Serializable {
   Map<String, Object?> toJson() => {Field.arguments: args};
 }
 
+/// Errors that the Analysis Server returns for failed argument edits.
+///
+/// These should be kept in sync with those listed in:
+/// pkg/analysis_server/lib/src/lsp/constants.dart
```

Comment:
can we have a test that covers this?

Reply:
Hmm I was trying to think how we could test this. We can't pull in analysis_server as a dependency (otherwise I would just use the error definitions there instead) so I'm not sure how we could have a test to check they are in sync. Any ideas?

Reply:
can we load the raw content of this file from GitHub and parse it? A bit hacky but still could be worth it to have a way to catch changes

Reply:
Filed https://github.com/flutter/devtools/issues/8824 and added TODOs